src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -61,6 +60,13 @@ bool ClientRequest::is_pg_op() const
     [](auto& op) { return ceph_osd_op_type_pg(op.op.op); });
 }
 
+void ClientRequest::may_set_prev_op()
+{
+  if (__builtin_expect(!prev_op_id, true)) {
```

Comment:
could you be more specific, like
```c++
// set prev_op_id if it's not set yet
if (__builtin_expect(prev_op_id == 0, true)) {
```
for two reasons:

1. `0` is a magic number, but we can make it more explicit 
2. a comment helps other developers understand the intention here.

Reply:
@tchaikov done:-)